DNN Platform/Library/Entities/Tabs/TabVersions/TabVersionController.cs

```diff
@@ -112,6 +117,26 @@ private static string GetTabVersionsCacheKey(int tabId)
         {
             return string.Format(DataCache.TabVersionsCacheKey, tabId);
         }
+        
+        protected virtual TimeZoneInfo GetDatabaseDateTimeOffset()
```

Comment:
One last comment. I think this method should be renamed to `GetDatabaseTimeZone` and moved. Maybe also to `DateUtils`?